CHANGES.rst

```diff
@@ -128,6 +128,14 @@ migration of the IO plugins to `RosettaSciIO
   and ``metadata_keys`` keywords were removed. Use ``dataset_key``, ``dataset_path``
   and ``metadata_key`` instead.
 
+Data Visualisation
```

Comment:
```suggestion
Data Visualization
```

Reply:
We're not very consistent on this, but visualize/visualization is more frequent than visualise/visualisation